test/JDBC/expected/PARTITION-vu-prepare.out

```diff
@@ -0,0 +1,458 @@
+GO
+
+-- identifier length greater than 64
+CREATE PARTITION SCHEME PartitionSchemeNameGreaterThan64AndLessThan128abcdefghijklmnopqrstuvwxyz AS
+PARTITION PartitionFunctionNameGreaterThan64AndLessThan128abcdefghijklmnopqrstuvwxyz 
+ALL TO ([PRIMARY]);
+GO
+
+--------------------------------------------------
+--- name identifiers test
+--------------------------------------------------
+CREATE PARTITION FUNCTION PARTITION_FUNCTIONログインαιώνια (int)
+AS RANGE RIGHT FOR VALUES (0, 500, 1000);
+GO
+
+CREATE PARTITION SCHEME PARTITION_SCHEMEログインαιώνια AS
+PARTITION PARTITION_FUNCTIONログインαιώνια
+ALL TO ([PRIMARY]);
+GO
+
+CREATE PARTITION FUNCTION [PARTITION_FUNCTION  유니코드스키마👻] (int)
+AS RANGE RIGHT FOR VALUES (0, 500, 1000);
+GO
+
+CREATE PARTITION SCHEME [PARTITION_SCHEME  유니코드스키마👻] AS
+PARTITION [PARTITION_FUNCTION  유니코드스키마👻]
+ALL TO ([PRIMARY]);
+GO
+
+
+-- qouted identifer
+SET QUOTED_IDENTIFIER ON;
+GO
+
+CREATE PARTITION FUNCTION "PARTITION_FUNCTION 😎$@ #123 🌍rder" (int)
+AS RANGE RIGHT FOR VALUES (0, 500, 1000);
+GO
+
+CREATE PARTITION SCHEME "PARTITION_SCHEME 😎$@ #123 🌍rder" AS
+PARTITION "PARTITION_FUNCTION 😎$@ #123 🌍rder"
+ALL TO ([PRIMARY]);
+GO
+
+SET QUOTED_IDENTIFIER OFF;
+Go
+
+
+--------------------------------------------------
+--- TO test Permission
+--------------------------------------------------
+CREATE DATABASE PartitionDb;
+GO
+
+USE PartitionDb;
+GO
+
+CREATE PARTITION FUNCTION PartitionDb_PartitionFunction (int)
+AS RANGE RIGHT FOR VALUES (500, 1000, 10000);
+Go
+
+CREATE PARTITION SCHEME PartitionDb_PartitionScheme AS
+PARTITION PartitionDb_PartitionFunction
+ALL TO ([PRIMARY]);
+GO
+
+USE MASTER
+go
+
+---------------------------------------------------------------
+--- CREATE/DROP Partition Function/Scheme inside Procedure
+---------------------------------------------------------------
+CREATE PROCEDURE PartitionProcedureCreatePartitionFunction AS
+BEGIN
+CREATE PARTITION FUNCTION PartitionFunctionInsideProc1 (int)
+AS RANGE RIGHT FOR VALUES (500, 1000, 10000)
+END;
+Go
+
+CREATE PROCEDURE PartitionProcedureCreatePartitionScheme AS
+BEGIN
+CREATE PARTITION SCHEME PartitionSchemeInsideProc1 AS
+PARTITION PartitionFunctionInsideProc1
+ALL TO ([PRIMARY])
+END;
+GO
+
+
+CREATE PROCEDURE PartitionProcedureCreatePartitionFunctionAndScheme AS
+BEGIN
+CREATE PARTITION FUNCTION PartitionFunctionInsideProc2 (int)
+AS RANGE RIGHT FOR VALUES (500, 1000, 10000);
+CREATE PARTITION SCHEME PartitionSchemeInsideProc2 AS
+PARTITION PartitionFunctionInsideProc2
+ALL TO ([PRIMARY]);
+END;
+Go
+
+CREATE PROCEDURE PartitionProcedureDropPartitionFunction AS
+BEGIN
+DROP PARTITION FUNCTION PartitionFunctionInsideProc1
+END;
+Go
+
+CREATE PROCEDURE PartitionProcedureDropPartitionScheme AS
+BEGIN
+DROP PARTITION SCHEME PartitionSchemeInsideProc1 
+END;
+GO
+
+CREATE PROCEDURE PartitionProcedureDropPartitionFunctionAndScheme AS
+BEGIN
+DROP PARTITION SCHEME PartitionSchemeInsideProc2;
+DROP PARTITION FUNCTION PartitionFunctionInsideProc2;
+END;
```

Comment:
Have we made sure that ddl export works?

Reply:
I have validated this manually and we will add requirred DDL scripting test in separate task.